1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -14,4 +14,8 @@ describe("fizzbuzz", () => {
         expect(fizzBuzz(5)).toBe('string')
     })
 
+    it('returns fizz when given 3', () => {
```

Comment:
NIT: Just a quick thought - it might be a good idea to change the description to Fizz to match the capitalized output of Fizz. Just a small suggestion to help make things more consistent and clear.